app/Http/Controllers/TaskController.php

```diff
@@ -15,17 +15,10 @@ public function index(){
     }
     
     public function store(Request $req){
```

Comment:
this function is missing `JavaDocs` you need to generate this component as per the docs: https://laravel.com/docs/9.x/controllers#resource-controllers